src/sdp/ZLIBTextCompressor.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const pako = require('pako');
+import Logger from '../logger/Logger';
+
+/**
+ * [[ZLIBTextCompressor]] Performs the text compression and decompression using zlib
+ */
+export default class ZLIBTextCompressor {
```

Comment:
thank you for not making this `TextCompressor` + `DefaultTextCompressor` lmao.

Reply:
Yeah...felt like an overkill for a simple class.

Reply:
For context, it used to be a build enforced rule to follow that pattern lol